core/src/main/java/hudson/scm/ChangeLogSet.java

```diff
@@ -205,6 +205,17 @@ public long getTimestamp() {
         @Exported
         public abstract User getAuthor();
 
+        /**
+         * Get from commit additional information
```

Comment:
It is unclear what this is for. `getMsg` is the commit message, so what is this? The Javadoc does not help.